### DISCUSSION TITLE
NumberInput automatically rounding off the number

### DISCUSSION THREAD

**Discussion by ghost** [mantinedev/mantine#3158] [Q&A] [answered]
Greeting all
I want to take the percentage as input, when I enter 2.2 it's not respecting the .2 and converting it to 2, I don't wish to round off the input. I read the document I didn't find anything.

**Comment by rtivital** [comment, marked as answer]
See https://mantine.dev/core/number-input/#decimal-steps

**Reply by ghost** [reply]
Thanks for the quick reply bro. and sorry for the silly question.
